2/두정수사이의합.js

```diff
@@ -0,0 +1,11 @@
+function solution(a, b) {
+  let answer = 0;
+  let c = 0;
+  if (a > b) {
+    c = a;
+    a = b;
+    b = c;
+  }
+  for (let i = a; i <= b; i++) answer += i;
```

Comment:
```suggestion
  }
  
  for (let i = a; i <= b; i++) {
    answer += i;
  }
```
1. 여기도 마찬가지로 구문과 구문 사이는 줄바꿈 해줍니다.
2. if문을 제외한 모든 제어문(i.e. if, else, for, do, while..)은 하나의 구문만 포함하더라도 중괄호를 모두 해주는게 좋습니다.